src/main/java/org/jabref/gui/help/AboutDialogViewModel.java

```diff
@@ -8,7 +8,6 @@
 import javafx.beans.property.ReadOnlyBooleanWrapper;
 import javafx.beans.property.ReadOnlyStringProperty;
 import javafx.beans.property.ReadOnlyStringWrapper;
-
```

Comment:
There should be an empty line here, which is also one of the reasons why the travis build fails.